files/images/teamspeak_viewer/badges/addon_author_gold_details.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 64 64"><title>gold-silver-bronze-badge</title><g id="Layer_1" data-name="Layer 1"><path d="M7.52,6.11A53.85,53.85,0,0,1,32,0,54.59,54.59,0,0,1,56.47,6.11l.93.46v31l-.06.43a41.44,41.44,0,0,1-9.93,17.86C42.43,61.06,36.65,64,32,64s-10.19-3.39-15.08-8.5A42.23,42.23,0,0,1,6.66,38l-.06-.43v-31l.92-.46ZM32,3.33A50.17,50.17,0,0,0,9.95,8.64V37.36a38.93,38.93,0,0,0,9.37,15.83c4.3,4.49,9.09,7.47,12.68,7.47,3.79,0,8.66-2.57,13-7.1a38,38,0,0,0,9.05-16.19V8.65A50.87,50.87,0,0,0,32,3.33Z" style="fill:#e0bf5e;fill-rule:evenodd"/><path d="M31.68,27.86a7.1,7.1,0,1,0-5-2.08,7.08,7.08,0,0,0,5,2.08Zm7.43.32a10.5,10.5,0,1,1,3.08-7.43A10.47,10.47,0,0,1,39.11,28.19Z" style="fill:#e0bf5e;fill-rule:evenodd"/><path d="M31.8,31.25a9.73,9.73,0,0,0-7.62,4.17,17.55,17.55,0,0,0-3.36,10.65h-3.4a21,21,0,0,1,4.07-12.71A13.07,13.07,0,0,1,31.8,27.85Z" style="fill:#e0bf5e;fill-rule:evenodd"/><path d="M42.78,46.07c0-.51,0-1-.05-1.46s-.09-1-.16-1.43l3.36-.49q.12.82.18,1.68c0,.58.06,1.14.06,1.7Z" style="fill:#e0bf5e;fill-rule:evenodd"/><path d="M41.12,40.71,32.83,49h0l-.07.07h0l0,0-.12.11a4.82,4.82,0,0,1-1.3.78h0a3.93,3.93,0,0,1-4.51-1.12,4.34,4.34,0,0,1-1.16-2.77,3.31,3.31,0,0,1,1.18-2.91l0,0h0L35.1,35a7.09,7.09,0,0,1-.19-1.67,8.49,8.49,0,0,1,1.51-4.69,8.91,8.91,0,0,1,3.69-3.15,6.83,6.83,0,0,1,2.77-.6,8,8,0,0,1,3.22.67l2.26,1L46.62,28.3,43.2,31.75h0l-.31.31a.46.46,0,0,0-.1.14h0a.45.45,0,0,0,0,.17.46.46,0,0,0,.14.33.45.45,0,0,0,.32.13.48.48,0,0,0,.18,0,.46.46,0,0,0,.14-.1h0l3.82-3.82L49,27.22l1.05,2.12a7.94,7.94,0,0,1,.83,3.56,7.84,7.84,0,0,1-.41,2.49,8.25,8.25,0,0,1-2.76,3.88,7.77,7.77,0,0,1-4.81,1.63,8.62,8.62,0,0,1-1.79-.19ZM30.43,46.65l9-9,.74-.75,1,.34a5.56,5.56,0,0,0,1.76.31,4.42,4.42,0,0,0,2.73-.93,4.83,4.83,0,0,0,1.61-2.27,4.69,4.69,0,0,0,.18-.72l-1.52,1.52h0a3.85,3.85,0,0,1-6.27-4.2h0a3.86,3.86,0,0,1,.83-1.24h0l.31-.31.83-.84-.16.07a5.47,5.47,0,0,0-2.25,1.94,5.08,5.08,0,0,0-.93,2.79,3.55,3.55,0,0,0,.3,1.45l.47,1.06-.83.82-9,9h0a.6.6,0,0,0-.21.53,1,1,0,0,0,.29.49.54.54,0,0,0,.75.25h0a1.45,1.45,0,0,0,.29-.15h0l.07-.07Z" style="fill:#e0bf5e;fill-rule:evenodd"/></g><g id="Ebene_1" data-name="Ebene 1"><rect y="63.83" width="0.17" height="0.17" style="fill:#8190aa;opacity:0.01"/><rect x="63.83" y="63.83" width="0.17" height="0.17" style="fill:#8190aa;opacity:0.01"/><rect width="0.17" height="0.17" style="fill:#8190aa;opacity:0.01"/><rect x="63.83" width="0.17" height="0.17" style="fill:#8190aa;opacity:0.01"/><rect width="0.17" height="0.17" style="fill:#8190aa;opacity:0.01"/><rect x="63.83" width="0.17" height="0.17" style="fill:#8190aa;opacity:0.01"/></g></svg>
```

Comment:
Why are there no line breaks here?

Reply:
Rechecked with original source. The file is untouched from original.